localization/pose_estimator/ndt_scan_matcher/ndt_scan_matcher/README.md

```diff
@@ -0,0 +1,61 @@
+# ndt scan matcher
```

Comment:
```suggestion
# ndt_scan_matcher
```
Nit: Should be snake_case. Please check the merge guidelines <https://tier4.atlassian.net/wiki/spaces/ATWROSS/pages/2163574164>.

Reply:
Thank you!